### ISSUE TITLE
Add ability to switch between standard layout and gallery theme in CMS

### ISSUE THREAD

**Issue by MattyBalaam** [colymba/GridFieldGalleryTheme#18]
It would be handy to include the ability for CMS authors to toggle between the standard and gallery views.

**Comment by colymba** [comment]
Indeed! This can probably be solved with a simple tickbox that toggles the gallery theme css class.... maybe...

**Comment by colymba** [comment]
Note to self: Let's add in a slider to change the thumbnail size. Something like https://github.com/darthrubens/radios-to-slider could work.
